### PR TITLE
systemd-graceful-shutdown: copy edit

### PR DESCRIPTION
**Pull request by frasertweedale**
Copy editing the post for grammar, spelling and style.

### CODE REVIEW

**Comment by frasertweedale** [comment]
Note that I haven't built and inspected the resulting HTML - only modified the markdown in place.  If I made some error or you wish to discuss anything further, please do not hesitate.

File: content/stopping-systemd-in-openshift.md [hunk 1]
```diff
@@ -280,9 +277,9 @@ spec:
 ```
 
 - (1) The lifecycle hooks for that container.
-- (2) Indicate that will be called before stopping the container.
-- (3) It will be an exec command.
-- (4) The command to be executed; the command should exist inside the container.
+- (2) A `preStop` hook is called before stopping the container.
+- (3) It will be an `exec` command.
+- (4) The command to be executed; the executable must exist in the container.
```
Comment (frasertweedale) [inline content/stopping-systemd-in-openshift.md:283]: Note that the rendered HTML for this list is a bit strange (extra space beween some of the bullets).  But I decided not to try and fix that.  Just giving you the heads-up.

Comment (avisiedo) [reply content/stopping-systemd-in-openshift.md:283]: Thanks @frasertweedale ; Pelican is not rendering this as I was expecting; I have tried with cobalt and that is working better when rendering the resulting html. I will move to [cobalt](https://cobalt-org.github.io) as soon as possible.

**Comment by avisiedo** [comment]
@frasertweedale thank you very much for this review; I appreciate it a lot. Merging.

**Pull request merged**